eng/scripts/Language-Settings.ps1

```diff
@@ -340,8 +371,34 @@ function UpdateDocsMsPackages($DocConfigFile, $Mode, $DocsMetadata) {
     $outputPackages += @{ name = $packageName }
   }
 
-  $packageConfig.npm_package_sources = $outputPackages
+  $packageValidation = ValidatePackagesForDocs $outputPackages
+  $validationHash = @{}
+  foreach ($result in $packageValidation) {
+    $validationHash[$result.Package.name] = $result
+  }
+
+  # Remove invalid packages
+  $finalOutput = @()
+  foreach ($package in $outputPackages) {
+    if (!$validationHash[$package.name].Success) {
+      LogWarning "Removing invalid package: $($package.name)"
```

Comment:
Should we remove or simply not update the package? I'd suggest we keep whatever package we had before so that we don't start deleting the old docs if a new version has a issue.